sdk/src/connections/network/network_depth_sensor.cpp

```diff
@@ -435,6 +459,18 @@ aditof::Status NetworkDepthSensor::writeAfeRegisters(const uint16_t *address,
                                                      size_t length) {
     using namespace aditof;
 
+    if (address == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters address null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    if (data == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters data null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    assert(length > 0);
```

Comment:
`length` has the type `size_t` which by definition is unsigned, so this is not needed.

Reply:
In this way we will prevent the length to be set to 0.